### PR TITLE
[EDU-4077] feat: add usage guides for console kit

### PR DESCRIPTION
**Pull request by bru-andrade**
### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+description: aaaa
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/customize-console-kit/
+---
+
+The Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
+
+:::caution[warning]
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx:12]: Is it necessary to deploy the project to follow this guide? 

Coming from the CLI perspective:

- User will link the project to azion `azion link`
- The link commands adapts the project to azion
- The user can run it locally (no deployment required) 
- If the user wants to deploy he can do so, but it's optional.

File: src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and [deploy the project using the Azion CLI](/en/documentation/devtools/console-kit/#deploying-the-console-kit-to-the-edge). After you clone the repository, you can work on changing the forked version of the Azion Console and deploy the project to the edge using the CLI.
+:::
+
+---
+
+## Add UI component as a callout to direct users to another page
+
+This example, you'll add a new UI component to a list view to draw attention an important step of a workflow process for Console users. In this case, you'll add the `InlineMessage` and `PrimeButton` components to the **Edge Applications** list view to alert your user that they must create a domain to see edge applications online, with a button action that directs users to the `create-domain` route. To do so:
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx:20]:
```suggestion
In this example, you'll add a new UI component to a list view to draw attention to an important step of a workflow process for Console users. In this case, you'll add the `InlineMessage` and `PrimeButton` components to the **Edge Applications** list view to alert your users that they must create a domain to see edge applications online, with a button action that directs users to the `create-domain` route. To do so:
```

File: src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/customize-console-kit/
+---
+
+The Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
+
+:::caution[warning]
+To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and [deploy the project using the Azion CLI](/en/documentation/devtools/console-kit/#deploying-the-console-kit-to-the-edge). After you clone the repository, you can work on changing the forked version of the Azion Console and deploy the project to the edge using the CLI.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx:13]:
```suggestion
To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and initialize the project using the Azion CLI. After you clone the repository, you can work on changing the forked version of the Azion Console and, if you wish, deploy the project to the edge using the CLI.
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+  // import statement for the PrimeButton component
+  import Illustration from '@/assets/svg/illustration-layers.vue'
+
+  // import statement for the router function
+  import { useRouter } from 'vue-router'
+  ...
+```
+
+4. In the const declaration section, create the `router` const to call the router function:
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:43]:
```suggestion
4. In the const declaration section, create the `router` const to instantiate the router function:
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  ...
+```
+
+4. In the const declaration section, create the `router` const to reference the router function:
+
+```js title="src/views/EdgeApplications/ListView.vue"
+  ...
+  defineOptions({ name: 'list-edge-applications' })
+
+  // instantiate the router function for the button action
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:49]:
```suggestion
  // reference the router function for the button action
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 6]
```diff
@@ -0,0 +1,106 @@
+---
+title: How to customize the Azion Console UI using the Console Kit
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:2]:
```suggestion
title: How to customize the Azion Console interface using the Console Kit
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_ui
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/customize-ui-console-kit/
+---
+
+The Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
+
+:::caution[warning]
+To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and initialize the project using the Azion CLI to run it locally. Then, you can work on changing the forked version of the Azion Console and, once you're done, deliver the project to the edge using the CLI.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:13]:
```suggestion
To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and initialize the project using Azion CLI to run it locally. Then, you can work on changing the forked version of the Azion Console and, once you're done, deliver the project to the edge using the CLI.
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+:::
+
+---
+
+## Adding a UI component and creating a route function for navigation
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:18]:
```suggestion
## Adding a interface component and creating a route function for navigation
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+In this example, you'll add a new UI component to a list view to draw attention to an important step of a workflow process for Console users.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:20]:
```suggestion
In this example, you'll add a new user interface (UI) component to a list view to draw attention to an important step of a workflow process for Console users. 
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and initialize the project using the Azion CLI to run it locally. Then, you can work on changing the forked version of the Azion Console and, once you're done, deliver the project to the edge using the CLI.
+:::
+
+---
+
+## Adding a UI component and creating a route function for navigation
+
+In this example, you'll add a new UI component to a list view to draw attention to an important step of a workflow process for Console users. 
+
+In this case, you'll add the `InlineMessage` and `PrimeButton` components to the **Edge Applications** list view to alert your users that they must create a domain to see edge applications online, with a button action that directs users to the `create-domain` route. To do so:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:22]:
```suggestion
You'll add the `InlineMessage` and `PrimeButton` components to the **Edge Applications** list view to alert your users that they must create a domain to see edge applications online, with a button action that directs users to the `create-domain` route. To do so:
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+
+  // import statement for the PrimeButton component
+  import Illustration from '@/assets/svg/illustration-layers.vue'
+
+  // import statement for the router function
+  import { useRouter } from 'vue-router'
+  ...
+```
+
+4. In the const declaration section, create the `router` const to reference the router function:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:43]:
```suggestion
4. In the `const declaration` section, create the `router` const to reference the router function:
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:43]: or bold*

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+```js title="src/views/EdgeApplications/ListView.vue"
+  ...
+  defineOptions({ name: 'list-edge-applications' })
+
+  // reference the router function for the button action
+  const router = useRouter()
+  ...  
+```
+
+5. Create the `navigateToDomains` function that will be executed when the user clicks the `PrimeButton` before closing the `<script>` tag:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:54]:
```suggestion
5. Create the `navigateToDomains` function that will be executed when the user clicks the `PrimeButton` before closing the `<script>` tag:
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+          Domain
+        </PrimeButton>
+         and select an edge application to launch online.
+      </InlineMessage>
+    </template>
+    <template #content>
+    ...
+```
+
+7. On the browser, access the localhost address and navigate to the **Edge Applications** page. You should see the `InlineMessage` element under the heading.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:97]:
```suggestion
7. On the browser, access the localhost address and navigate to the **Edge Applications** page. You should see the `InlineMessage` element with a link under the heading.
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 14]
```diff
@@ -0,0 +1,106 @@
+---
+title: How to customize the Azion Console UI using the Console Kit
+description: The Azion Console Kit allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:3]:
```suggestion
description: Azion Console Kit allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
```

File: src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx [hunk 15]
```diff
@@ -0,0 +1,93 @@
+---
+title: How to integrate the Azion Console and the Azion API using the Console Kit
+description: The Azion Console Kit allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to show more information from the Azion API on the Console.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx:3]:
```suggestion
description: Azion Console Kit allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to show more information from the Azion API on the Console.
```

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_ui
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/customize-ui-console-kit/
+---
+
+The Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:10]:
```suggestion
Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
```

File: src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_api
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/integrate-with-api-console-kit/
+---
+
+The Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to show more information from the Azion API on the Console.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx:10]:
```suggestion
Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to show more information from the Azion API on the Console.
```

File: src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+
+:::caution[warning]
+To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and initialize the project using the Azion CLI to run it locally. Then, you can work on changing the forked version of the Azion Console and, once you're done, deliver the project to the edge using the CLI.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx:13]:
```suggestion
To follow this guide, you must first [setup your Console Kit project](/en/documentation/devtools/console-kit/#setting-up-a-project) on a forked repository in your account and initialize the project using Azion CLI to run it locally. Then, you can work on changing the forked version of the Azion Console and, once you're done, deliver the project to the edge using the CLI.
```

File: src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+        header: 'Edge Cache TTL'
+      }
+    ]
+  })
+```
+
+6. On the browser, access the localhost address and navigate to the **Edge Applications** page.
+7. Create an edge application or select an existing one.
+8. Navigate to the **Cache Settings** tab. You should now see the new columns appear in the UI.
+9. After you're done, run `azion deploy` to launch the changes to the edge.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/devtools/integrate-with-api/index.mdx:83]:
```suggestion
5. On the browser, access the localhost address and navigate to the **Edge Applications** page.
6. Create an edge application or select an existing one.
7. Navigate to the **Cache Settings** tab. You should now see the new columns appear in the UI.
8. After you're done, run `azion deploy` to launch the changes to the edge.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 20]
```diff
@@ -0,0 +1,107 @@
+---
+title: Como personalizar a UI do Console Azion usando o Console Kit
+description: O Azion Console Kit permite que você crie uma interface Azion personalizada para atender às necessidades do seu negócio. Neste guia, você aprenderá a fazer algumas alterações visuais simples no Console Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:3]:
```suggestion
description: O Azion Console Kit permite que você crie uma interface Azion personalizada para atender às necessidades do seu negócio. Neste guia, você aprenderá a fazer algumas alterações visuais simples no Console da Azion.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_ui
+menu_namespace: devtoolsMenu
+permalink: /documentacao/produtos/guias/personalizar-ui-console-kit/
+---
+
+O **Console Kit** da Azion permite que você crie uma interface Azion personalizada para atender às necessidades do seu negócio. Neste guia, você aprenderá a fazer algumas alterações visuais simples no Console Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:10]:
```suggestion
O **Console Kit** da Azion permite que você crie uma interface Azion personalizada para atender às necessidades do seu negócio. Neste guia, você aprenderá a fazer algumas alterações visuais simples no Console da Azion.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 22]
```diff
@@ -0,0 +1,107 @@
...
+
+:::caution[warning]
+Para seguir este guia, você deve primeiro [configurar seu projeto Console Kit](/pt-br/documentacao/devtools/console-kit/#configure-um-projeto) em um repositório bifurcado na sua conta e inicializar o projeto usando o CLI da Azion para executá-lo localmente. Então, você pode trabalhar na alteração da versão bifurcada do Console Azion e, uma vez concluído, entregar o projeto na edge usando o CLI.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:13]:
```suggestion
Para seguir este guia, você deve primeiro [configurar seu projeto do Console Kit](/pt-br/documentacao/devtools/console-kit/#configure-um-projeto) em um repositório bifurcado (forked) na sua conta e inicializar o projeto usando a CLI da Azion para executá-lo localmente. Então, você pode trabalhar na alteração da versão bifurcada do Console da Azion e, uma vez concluído, entregar o projeto no edge usando a CLI.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 23]
```diff
@@ -0,0 +1,107 @@
...
+:::
+
+---
+
+## Adicionar um componente de UI e criar uma função de route para navegação
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:18]:
```suggestion
## Adicionar um componente de interface e criar uma função de route para navegação
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 24]
```diff
@@ -0,0 +1,107 @@
...
+
+Neste exemplo, você adicionará um novo componente de UI a uma list view para chamar a atenção para uma etapa importante de um processo de fluxo de trabalho para os usuários do Console.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:20]:
```suggestion
Neste exemplo, você adicionará um novo componente de onterface de usuário (UI) a uma list view para chamar a atenção para uma etapa importante de um processo de fluxo de trabalho para os usuários do Console.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 25]
```diff
@@ -0,0 +1,107 @@
...
+
+Neste caso, você adicionará os componentes `InlineMessage` e `PrimeButton` à list view de **Edge Applications** para alertar seus usuários de que eles devem criar um domínio para entregar edge applications online, com uma ação de botão que direciona os usuários para a rota `create-domain`. Para fazer isso:
+
+1. Abra o projeto Console Kit na sua IDE.
+2. Execute `azion dev` para inicializar um servidor de desenvolvimento local.
+3. Na pasta `src/views`, localize a list view de Edge Application e modifique o código conforme a seguir:
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:27]:
```suggestion
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 26]
```diff
@@ -0,0 +1,107 @@
...
+---
+
+## Adicionar um componente de UI e criar uma função de route para navegação
+
+Neste exemplo, você adicionará um novo componente de UI a uma list view para chamar a atenção para uma etapa importante de um processo de fluxo de trabalho para os usuários do Console.
+
+Neste caso, você adicionará os componentes `InlineMessage` e `PrimeButton` à list view de **Edge Applications** para alertar seus usuários de que eles devem criar um domínio para entregar edge applications online, com uma ação de botão que direciona os usuários para a rota `create-domain`. Para fazer isso:
+
+1. Abra o projeto Console Kit na sua IDE.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:24]:
```suggestion
1. Abra o projeto do Console Kit na sua IDE.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 27]
```diff
@@ -0,0 +1,107 @@
...
+          Domain
+        </PrimeButton>
+         and select an edge application to launch online.
+      </InlineMessage>
+    </template>
+    <template #content>
+    ...
+```
+
+7. No navegador, acesse o localhost e navegue até a página **Edge Applications**. Você deve ver o elemento `InlineMessage` abaixo do título.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:98]:
```suggestion
7. No navegador, acesse o localhost e navegue até a página **Edge Applications**. Você deve ver o elemento `InlineMessage` com um link abaixo do título.
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 28]
```diff
@@ -0,0 +1,107 @@
...
+8. Depois de terminar, execute `azion deploy` para lançar as alterações na edge.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:99]:
```suggestion
8. Depois de terminar, execute `azion deploy` para lançar as alterações no edge.
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 29]
```diff
@@ -0,0 +1,93 @@
+---
+title: Como integrar o Console Azion e a API da Azion usando o Console Kit
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:2]:
```suggestion
title: Como integrar o Console da Azion e a API da Azion usando o Console Kit
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 30]
```diff
@@ -0,0 +1,93 @@
...
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_api
+menu_namespace: devtoolsMenu
+permalink: /documentacao/produtos/guias/integrar-com-api-console-kit/
+---
+
+O **Console Kit** da Azion permite que você crie uma interface Azion personalizada para atender às necessidades do seu negócio. Neste guia, você aprenderá a mostrar mais informações da API da Azion no Console.
+
+:::caution[warning]
+Para seguir este guia, você deve primeiro [configurar seu projeto Console Kit](/en/documentation/devtools/console-kit/#setting-up-a-project) em um repositório bifurcado na sua conta e inicializar o projeto usando o CLI da Azion para executá-lo localmente. Então, você pode trabalhar na alteração da versão bifurcada do Console da Azion e, uma vez concluído, entregar o projeto para a edge usando o CLI.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:13]:
```suggestion
Para seguir este guia, você deve primeiro [configurar seu projeto do Console Kit](/en/documentation/devtools/console-kit/#setting-up-a-project) em um repositório bifurcado (forked) na sua conta e inicializar o projeto usando a CLI da Azion para executá-lo localmente. Então, você pode trabalhar na alteração da versão bifurcada do Console da Azion e, uma vez concluído, entregar o projeto no edge usando a CLI.
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 31]
```diff
@@ -0,0 +1,93 @@
...
+:::
+
+Para integrar *APIs externas* com a interface do Console da Azion, você pode criar um novo serviço Axios API em `src/services/axios`, adaptar cada serviço conforme necessário e seguir as etapas abaixo.
+
+---
+
+## Obter dados da API da Azion e mostrar na interface do Console
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:20]:
```suggestion
## Obtenha dados da API da Azion e mostre na interface do Console
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 32]
```diff
@@ -0,0 +1,93 @@
...
+
+Neste exemplo, você modificará um serviço no Console para mostrar dois campos adicionais da API da Azion na tabela de configurações de cache da aplicação edge.
+
+O objetivo é mostrar os valores TTL para cache do navegador e cache da edge, informados pelas propriedades `browser_cache_settings_maximum_ttl` e `cdn_cache_settings_maximum_ttl` na [API de Cache Settings](https://api.azion.com/#fc4c8368-a64d-4743-9064-58e03786edf4). Para fazer isso:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:24]:
```suggestion
O objetivo é mostrar os valores TTL para cache do navegador e cache do edge, informados pelas propriedades `browser_cache_settings_maximum_ttl` e `cdn_cache_settings_maximum_ttl` na [API de Cache Settings](https://api.azion.com/#fc4c8368-a64d-4743-9064-58e03786edf4). Para fazer isso:
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+
+1. Abra o projeto Console Kit no seu IDE.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:26]:
```suggestion
1. Abra o projeto do Console Kit no seu IDE.
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 34]
```diff
@@ -0,0 +1,93 @@
...
+  }))
+
+  return {
+    body: parseHttpResponse,
+    statusCode: httpResponse.statusCode
+  }
+}
+```
+
+5. Agora localize a **Cache Settings** list view e modifique-a da seguinte forma:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:49]:
```suggestion
4. Agora localize a list view **Cache Settings** e modifique-a da seguinte forma:
```

File: src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx [hunk 35]
```diff
@@ -0,0 +1,93 @@
...
+        header: 'Edge Cache TTL'
+      }
+    ]
+  })
+```
+
+6. No navegador, acesse o endereço localhost e navegue até a página **Edge Applications**.
+7. Crie uma aplicação edge ou selecione uma existente.
+8. Navegue até a aba **Cache Settings**. Agora você deve ver as novas colunas aparecerem na UI.
+9. Depois de terminar, execute `azion deploy` para lançar as alterações para a edge.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/devtools/integrar-com-api/index.mdx:83]:
```suggestion
5. No navegador, acesse o endereço localhost e navegue até a página **Edge Applications**.
6. Crie uma edge application ou selecione uma existente.
7. Navegue até a aba **Cache Settings**. Agora você deve ver as novas colunas aparecerem na interface.
8. Depois de terminar, execute `azion deploy` para lançar as alterações no edge.
```

**Review by gabriel-alves-azion — APPROVED** [review]

File: src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx [hunk 36]
```diff
@@ -0,0 +1,106 @@
+---
+title: How to customize the Azion Console UI using the Console Kit
+description: The Azion Console Kit allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_ui
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/customize-ui-console-kit/
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/devtools/customize-console-ui/index.mdx:7]:
```suggestion
permalink: /documentation/products/guides/customize-interface-console-kit/
```

File: src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx [hunk 37]
```diff
@@ -0,0 +1,107 @@
+---
+title: Como personalizar a UI do Console Azion usando o Console Kit
+description: O Azion Console Kit permite que você crie uma interface Azion personalizada para atender às necessidades do seu negócio. Neste guia, você aprenderá a fazer algumas alterações visuais simples no Console Azion.
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit_ui
+menu_namespace: devtoolsMenu
+permalink: /documentacao/produtos/guias/personalizar-ui-console-kit/
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/devtools/customizar-console-ui/index.mdx:7]:
```suggestion
permalink: /documentacao/produtos/guias/personalizar-interface-console-kit/
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**